pyterrier/java.py

```diff
@@ -0,0 +1,171 @@
+from functools import wraps
+import os
+from typing import Callable, Optional
+from pyterrier import mavenresolver
+
+import pyterrier as pt
+
+
+_started = False
+
+
+def _legacy_post_init(jnius):
+    pt.HOME_DIR = pt.io.pyterrier_home()
+    pt.properties = autoclass('java.util.Properties')()
+    pt.ApplicationSetup = autoclass('org.terrier.utility.ApplicationSetup')
+    pt.autoclass = jnius.autoclass
+    pt.cast = jnius.cast
+
+
+@pt.utils.once()
+def init() -> None:
```

Comment:
should this not be private?

Reply:
I'd like a way to force init if you want to. E.g., you might want to be sure that java has started before timing stuff.